aweber.com.landing-pages-subdomain.json

```diff
@@ -9,7 +9,7 @@
    "variableDescription":"",
    "records":[
        {
-         "type": "A",
+         "type": "CNAME",
```

Comment:
CNAME on APEX is not allowed. If intended usage is only for subdomains, please set `hostRequired` to `true`.

Reply:
ah that's it. thanks

Reply:
So, best if I use a variable like %subdomain%:
`"host": "%subdomain%"`